apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts

```diff
@@ -314,3 +417,63 @@ test.describe("Verify send max user flow", () => {
     },
   );
 });
+
+for (const transaction of transactionAddressValid) {
+  test.describe("Send funds step 1 (Recipient) - positive cases (Button enabled)", () => {
```

Comment:
I think this should be a part of full e2e scenario where we actually send funds (even if without broadcasting) because creating test just for one check that button is enabled is an overkill